server/app/controllers/applicant/RedirectController.java

```diff
@@ -122,7 +122,7 @@ public CompletionStage<Result> programBySlug(Http.Request request, String progra
                                           .review(
                                               applicantId,
                                               programForExistingApplication.get().id()))
-                                  .flashing("redirected-from-program-slug", "true"));
+                                  .flashing("redirected-from-program-slug", programSlug));
```

Comment:
**suggestion:** consider using the session directly rather than the flash message: https://github.com/civiform/civiform/blob/main/server/app/controllers/CallbackController.java#L19

Reply:
Ack. Going to keep the existing design for now for expedience as we want to fix this issue for Seattle asap.